build.gradle

```diff
@@ -30,7 +30,14 @@ buildscript {
     }
 }
 
+plugins {
+    id "maven-publish"
+    id "digital.wup.android-maven-publish" version "3.6.2"
```

Comment:
If the project was using AGP 7.1 or newer we wouldn't need this dependency. Also, the scripts for attaching sources wouldn't be needed because we could simply use `withSourcesJar()` instead.

The AmityUIKit projects uses AGP 4.2.2 but I don’t think updating is a good idea. 
If we update AGP for AmityUIKit fork it might require us to migrate parts of the existing scripts which can be a lot of work.
Also, as Amity team updates their gradle scripts, they might not be compatible with the new AGP on our side so it can cause more work in the future when we’re pulling the new changes. Therefore I’ll leave it as is.